### DISCUSSION TITLE
プレゼント企画やめませんか？

### DISCUSSION THREAD

**Discussion by iamryosuke** [increments/qiita-discussions#951] [要望・アイデア・不具合]
アドベントカレンダーで記事を多く募るためか、プレゼント企画をやっていましたが、それをやると中身のない記事が増殖されて、サイトそのものの信頼性を落とすだけなんですがね。C# で明らかに生成AIによる記事では？っていうものが見受けられるので。

**Comment by tanaka-asami415** [comment]
@iamryosuke 
ご意見ありがとうございます。

協賛企業様との企画は、Qiitaのサービスを継続するために必要な取り組みでもあり完全に無くすことは難しいですが
課題点についても真摯に受け止め、多くの方に楽しんでいただけるよう引き続き改善を行っていきます🙇

ガイドライン違反等の記事は受賞対象に含めない等の対応も行っておりますので、問題のある記事を見つけられた際には、[通報機能](https://help.qiita.com/ja/articles/qiita-report) よりご報告いただけますと幸いです。
